security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -12,6 +12,17 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-22"]
+== OpenShift File Integrity Operator 0.1.22
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.22:
+
+* link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx OpenShift File Integrity Operator Bug Fix]
+
+[id="openshift-file-integrity-operator-0-1-22-bug-fixes"]
+=== Bug fixes
+* Previously, a system with a File Integrity Operator installed might interrupt the {product-title} upgrade, due to the  `/etc/kubernetes/aide.reinit` file.  This occurred if the `/etc/kubernetes/aide.reinit` file was present, but later removed prior to the `ostree` validation. In this release, `/etc/kubernetes/aide.reinit` was moved to the `/run` directory so that it does not conflict with the upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945274[*BZ#1945274*])
```

Comment:
* Previously, a system with a File Integrity Operator installed might interrupt the {product-title} upgrade, due to the  `/etc/kubernetes/aide.reinit` file.  This occurred if the `/etc/kubernetes/aide.reinit` file was present, but later removed prior to the `ostree` validation. With this update, `/etc/kubernetes/aide.reinit` is moved to the `/run` directory so that it does not conflict with the upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945274[*BZ#1945274*])

Reply:
Done